MAUI/TextInputLayout/Supported-input-views.md

```diff
@@ -0,0 +1,157 @@
+---
+layout: post
+title: Supported Input Views in .NET MAUI Text Input Layout | Syncfusion
+description: Learn here all about Supported Input Views support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
```

Comment:
description: Learn here all about Supported Input Views support in the Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.

Reply:
Addressed the changes